### DISCUSSION TITLE
Intellisense suggestions break the snippet flow

### DISCUSSION THREAD

**Discussion by liris8** [sleepymalc/VSCode-LaTeX-Inkscape#29] [Q&A] [answered]
The problem is that the suggestions from Intellisense intercede with my writing. This issue is that in many snippets you jump from one place to the other with the tab command. But this is the same command that triggers the Intellisense suggestions. See for example when I write the i and click tab this happens:
![Intellisense interceeding](https://github.com/sleepymalc/VSCode-LaTeX-Inkscape/assets/132703994/2639ffe8-1d7d-47d9-adc1-3c22a6ea4df9)
I don’t want to deactivate the suggestions because they are very useful for the json snippets, for the bib keys, the labels… I tried temporarily to change the suggestion time, but it was also inefficient. Furthermore, I know that you can escape the suggestions with the esc command, but also I find this annoying. So my question is, how do you guys manage this issue? If you have any useful advice for more efficient performance, I would be very grateful.

**Comment by sleepymalc** [comment, marked as answer]
I think you can look into `tab` in keybinding setting. For example, I disable this behavior:
```json
{
        "key": "tab",
        "command": "-acceptSelectedSuggestion",
        "when": "suggestWidgetHasFocusedSuggestion && suggestWidgetVisible && textInputFocus"
},
```
